src/fides/api/ops/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -258,7 +347,27 @@ def set_consent_preferences(
                     status_code=HTTP_400_BAD_REQUEST, detail=Pii(str(exc))
                 )
 
-    return _prepare_consent_preferences(db, provided_identity)
+    consent_preferences: ConsentPreferences = _prepare_consent_preferences(
+        db, provided_identity
+    )
+
+    # Note: This just queues the PrivacyRequest for processing
+    privacy_request_creation_results: Optional[
+        BulkPostPrivacyRequests
+    ] = queue_privacy_request_to_propagate_consent(
+        db,
+        provided_identity,
+        data.policy_key or DEFAULT_CONSENT_POLICY,
+        consent_preferences,
+    )
+
+    if privacy_request_creation_results:
+        consent_request.privacy_request_id = privacy_request_creation_results.succeeded[
+            0
+        ].id
+        consent_request.save(db=db)
```

Comment:
This is the primary change of this PR. In the existing PATCH `{{host}}/consent-request/{{consent_request_id}}/preferences` endpoint, After we save consent preferences, we queue a privacy request that attempts to propagate those preferences to third party systems.